Packages/manifest.json

```diff
@@ -1,5 +1,15 @@
 {
+    "scopedRegistries": [
+        {
+            "name": "MS Build for Unity",
+            "url": "https://pkgs.dev.azure.com/UnityDeveloperTools/MSBuildForUnity/_packaging/UnityDeveloperTools/npm/registry",
+            "scopes": [
+                "com.microsoft"
+            ]
+        }
+    ],
   "dependencies": {
+"com.microsoft.msbuildforunity": "0.8.1",
```

Comment:
```suggestion
    "com.microsoft.msbuildforunity": "0.8.1",
```

Reply:
Is this how it gets written in? Any way to correct the indentation?

Reply:
we can explicity write the indentation Unity's file currently supports. there is no functional difference with or without the indentation